src/libcore/macros.rs

```diff
@@ -255,6 +255,9 @@ macro_rules! debug_assert_ne {
 /// Helper macro for reducing boilerplate code for matching `Result` together
 /// with converting downstream errors.
 ///
+/// Prefer using `?` syntax to `try!`. `?` is built in to the language and is
+/// more succinct than `try!`, it is the standard method for error propogation.
```

Comment:
> `?` is built in to the language and is more succinct than `try!`, it is the standard method for error propogation.

These are two separate sentences and probably shouldn't be separated by a comma.